test_conformance/subgroups/subhelpers.h

```diff
@@ -1297,19 +1295,6 @@ template <typename Ty, typename Fns, size_t TSIZE = 0> struct test
             }
         }
 
-        for (std::string extension : test_params.required_extensions)
-        {
-            if (!is_extension_available(device, extension.c_str()))
-            {
-                log_info("The extension %s not supported on this device. SKIP "
-                         "testing - kernel %s data type %s\n",
-                         extension.c_str(), kname, TypeManager<Ty>::name());
-                return TEST_PASS;
-            }
-            kernel_sstr << "#pragma OPENCL EXTENSION " + extension
```

Comment:
What about pragmas. Do we need use them or not ?

Reply:
AFAICT the pragmas are not required by the extended subgroup specification. @bashbaug Can you confirm?

Reply:
Correct, no pragmas are (currently) required for the extended subgroups extensions.

Reply:
Thanks for confirming Ben!